TESTING.md

```diff
@@ -169,8 +169,102 @@ cargo make \
   test
 ```
 
-(`-r` tells testsys to also show the status of resources like the cluster and instances in addition to tests):
-
 ```shell
 cargo make watch-test
 ```
+
+## Migration Testing
+
+Migration testing starts testsys at a starting version, or a provided initial image id and migrates instances to the target version.
+In order to accomplish this a few artifacts need to be created:
+* `TUF` repos that are publicly accessible
+* A previous release of bottlerocket signed with available keys
+* The ami id from the previous release
+* Build images and repos for current changes
+
+### The setup
+
+#### Prepare `Infra.toml`
+
+A publicly available s3 bucket is needed to complete the next steps, once a bucket is created the steps may be continued.
```

Comment:
The example steps I provide include syncing tuf repos with the pre-update metadata. I was trying to make the process of onboarding to testsys as easy as possible by providing a sample workflow from start to finish. I can add to the beginning that it is not required to have, but this example will use it.

Reply:
I think it will be best to point whomever reads this guide to [this document](https://github.com/bottlerocket-os/bottlerocket/blob/develop/PUBLISHING.md#repo-location) so that they are aware that they are free to decide where to store their TUF repo. I would definitely remove the "publicly" available part since that isn't a good practice, and we shouldn't advocate for that. It will be best to say "use the URL of an accessible TUF repo"; and for your example, assume the contributor followed the link you will add, and that they have a URL that is on top of a properly configured S3 bucket (that's what ultimately happened in the document I linked).